core/dbt/include/global_project/macros/adapters/timestamps.sql

```diff
@@ -0,0 +1,44 @@
+{%- macro current_timestamp() -%}
+    {{ adapter.dispatch('current_timestamp', 'dbt')() }}
+{%- endmacro -%}
+
+{% macro default__current_timestamp() %}
+    current_timestamp
+{% endmacro %}
+
+{%- macro current_timestamp_in_utc() -%}
+    {{ adapter.dispatch('current_timestamp_in_utc', 'dbt')() }}
+{%- endmacro -%}
+
+{% macro default__current_timestamp_in_utc() %}
+    {{ default__current_timestamp() }} at time zone 'utc'
```

Comment:
IMO `default__current_timestamp_in_utc` should just be `default__current_timestamp` + `default__convert_timezone`

Even though the end result is basically the same on Postgres, it avoids the need to reimplement `current_timestamp_in_utc` if an adapter plugin has successfully reimplemented `current_timestamp` + `convert_timezone`